--- gotools/Makefile.am.orig	2015-03-25 03:50:31.000000000 +0800
+++ gotools/Makefile.am	2016-04-27 14:23:36.000000000 +0800
@@ -39,7 +39,7 @@ GOCFLAGS = $(CFLAGS_FOR_TARGET)
 GOCOMPILE = $(GOCOMPILER) $(GOCFLAGS)
 
 AM_LDFLAGS = -L $(libgodir) -L $(libgodir)/.libs
-GOLINK = $(GOCOMPILER) $(GOCFLAGS) $(AM_GOCFLAGS) $(LDFLAGS) $(AM_LDFLAGS) -o $@
+GOLINK = $(GOCOMPILER) $(GOCFLAGS) $(AM_GOCFLAGS) $(LDFLAGS) $(AM_LDFLAGS) -o $@ -static-libgo -static-libgcc -Wl,-dy -lgcc_s
 
 cmdsrcdir = $(srcdir)/../libgo/go/cmd
 
